Typechecker: Disallow free function redefinition (#9852)
ethereum/solidity: Changelog.md

### 0.7.2 (unreleased)

Important Bugfixes:

>  * Type Checker: Disallow two or more free functions with identical name and parameter types and a free function alias with the same name and parameter types as another free function.

Reviewer: ```suggestion
 * Type Checker: Disallow two or more free functions with identical name (potentially imported and aliased) and parameter types.
```
